nengo/builder/learning_rules.py

```diff
@@ -10,7 +10,88 @@
 from nengo.synapses import Lowpass
 
 
-class SimBCM(Operator):
+class IntermittentOperator(Operator):
+    """Base class for operators that support intermittent execution.
+
+    An intermittent operator allows for execution in less than each simulation
+    time step. To implement an intermittent operator derive from this class
+    and ensure that:
+
+    * In the constructor of the derived class you append to the operator lists
+      ``self.sets``, ``self.incs``, ``self.reads``, and ``self.updates`` or, if
+      you decide to overwrite them, include `step` in ``self.reads``.
+    * Set ``self.scaled`` to a list of signals that you want to scale by the
+      period length (``apply_every``) each time step.
+    * Instead of `make_step` implement `make_concrete_step` in the same way
+      you would implement `make_step` normally. No special case code is needed.
+
+    Parameters
+    ----------
+    apply_every : float or None
```

Comment:
This doesn't match the description in the learning rule operators, I think we should be consistent.  (I think yours is more clear than the one I wrote)